open-metadata-resources/open-metadata-deployment/ansible-egeria/roles/egeria-omag-server/templates/app_service.j2

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the ODPi Egeria project.
+
+[Unit]
+Description={{ app_name }}-{{ app_version }} service
+After=network.target
+
+[Service]
+
+UMask=0027
+User={{ app_user }}
+Group={{ app_group }}
+
+{% if app_service_environment %}
+{% for k, v in app_service_environment.items() %}
+Environment="{{ k }}={{ v }}"
+{% endfor %}
+{% endif %}
+
+WorkingDirectory={{ app_work_dir }}
+
+ExecStartPre=/bin/sh -c "rm -fR {{ app_tmp_dir }}/*"
+ExecStart=/bin/sh -c "/usr/bin/java\
```

Comment:
may be some call for running java at alt locations (tho this may be more in dev)